rpc/src/v1/traits/traces.rs

```diff
@@ -56,5 +56,9 @@ build_rpc_trait! {
 		/// Executes the transaction with the given hash and returns a number of possible traces for it.
 		#[rpc(name = "trace_replayTransaction")]
 		fn replay_transaction(&self, H256, TraceOptions) -> Result<TraceResults>;
+
+		/// Executes the transaction with the given hash and returns a number of possible traces for it.
```

Comment:
`Executes all the transactions at the given block and returns a number of possible traces for each transaction.`